joomla/joomla-cms
# [4.2] Fix redirect on login url error

Pull Request for Issue #38297.

### Summary of Changes
See https://github.com/joomla/joomla-cms/issues/38297 to understand the issue

### Testing Instructions
1. Install Joomla 4.2 nightly build
2. Confirm the issue https://github.com/joomla/joomla-cms/issues/38297
3. Apply patch, confirm that the issue fixed
4. If it is possible, test the login redirect on a multilingual website and make sure it is still working as expected.


### Actual result BEFORE applying this Pull Request
There is unnecessary lang variable with empty value in login, logout redirect URL


### Expected result AFTER applying this Pull Request
No unnecessary lang variable any,ore


### Documentation Changes Required
None
